nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/recurrent/SRU.java

```diff
@@ -43,16 +42,31 @@
     private SRUWeights weights;
 
     @Getter
-    private SDVariable mask;
+    private INDArray mask;
```

Comment:
Add a new `INDArray ndMask` field instead of replacing the `SDVariable mask` field.
We need the existing SDVariable functionality to remain.
Also don't modify the SDVariable constructor. The new INDArray constructors look OK though.